nsv13/code/modules/munitions/munitions.dm

```diff
@@ -1,3 +1,8 @@
+#define mstate_closed 0
+#define mstate_unscrewed 1
+#define mstate_unbolted 2
+#define mstate_priedout 3
```

Comment:
Defines should be in all-caps.
~~(Original review got deleted as I tried to unresolve it.)~~

Reply:
Not required by the engine.

Reply:
While not required, it's standard to have them in all caps. Also helps distinguish them from variables.

Example: Every other define in the code.